src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/SendConsentOrderInContestedSubmittedHandler.java

```diff
@@ -40,23 +67,60 @@ public boolean canHandle(CallbackType callbackType, CaseType caseType, EventType
     public GenericAboutToStartOrSubmitCallbackResponse<FinremCaseData> handle(FinremCallbackRequest callbackRequest,
                                                                               String userAuthorisation) {
 
-        FinremCaseDetails caseDetails = callbackRequest.getCaseDetails();
-        log.info("Invoking contested {} submitted callback for case id: {}", callbackRequest.getEventType(), caseDetails.getId());
+        FinremCaseDetails finremCaseDetails = callbackRequest.getCaseDetails();
+        FinremCaseData finremCaseData = finremCaseDetails.getData();
+        log.info("Invoking contested {} submitted callback for case id: {}", callbackRequest.getEventType(), finremCaseDetails.getId());
 
-        List<String> parties = generalOrderService.getParties(caseDetails);
-        log.info("Selected parties {} on case {}", parties, caseDetails.getId());
+        List<String> parties = generalOrderService.getParties(finremCaseDetails);
+        log.info("Selected parties {} on case {}", parties, finremCaseDetails.getId());
 
-        sendNotifications(callbackRequest, parties, userAuthorisation);
+        CaseDetails caseDetails = finremCaseDetailsMapper.mapToCaseDetails(finremCaseDetails);
+        ConsentOrderWrapper wrapper = finremCaseDetails.getData().getConsentOrderWrapper();
+        CaseDocument latestGeneralOrder = finremCaseData.getGeneralOrderWrapper().getGeneralOrderLatestDocument();
```

Comment:
is there any chance here      CaseDocument latestGeneralOrder = finremCaseData.getGeneralOrderWrapper();
latestGeneralOrder can be null?

Reply:
@rishikrsharma the latestGeneralOrder can be null here